src/tests/unit/tests/bug-filing/common/markup/html-formatter.test.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { It, Mock, MockBehavior } from 'typemoq';
+import { createFormatter } from '../../../../../../bug-filing/common/markup/html-formatter';
+
+describe('HTMLFormatter', () => {
+    const truncateMock = Mock.ofInstance((text: string) => text, MockBehavior.Strict);
```

Comment:
I think we should have this creation in a before each so we can be sure of no side-effects.

Reply:
updated